name convention bugfix (#55)
robotology/gazebo-fmi: Contributing.md

Contributing to gazebo-fmi
==========================

Code Style
----------
A set of conventions about how to write code for this project.

## Naming conventions
* Type names start with upper case.

> * Class member variabile names start with m_ then lower case.

Reviewer: ```suggestion
* Class member names start with `m_` followed by a lower case.
```

* Class member method names start with upper case.

> * Local variabile names start with lower case.

Reviewer: ```suggestion
* Local variable names start with lower case.
```

> For that not foreseen in the current document refer to:

Reviewer: ```suggestion
For any additional detail not covered by this document, please refer to
```
